sandboxes/andromeda/cannonfile.toml

```diff
@@ -0,0 +1,865 @@
+name = "andromeda-sandbox"
+version = "1"
+description = "Andromeda Perps Sandbox"
+
+[setting.owner]
+# Matches owner of core system that's provisions on Cannon network
+defaultValue = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266" # PK 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+# We can change the owner of pools and modules deployed in the Sandbox if we want
+#defaultValue = "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC" # PK 0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a
+description = "Hardhat/Anvil first test account"
+
+#
+#
+# TODO: We will need to use Pyth Mock, so these settings will change.
```

Comment:
Pyth mock module is available on cannon now
```
[import.pyth]
source = "pyth:1.4.0"
args = [120, 1]       # 120s, 1wei
```

Reply:
https://usecannon.com/packages/pyth/latest/13370-main

Reply:
Yeah this will come in a separate PR